### PR TITLE
Daji pink bed

### PR DESCRIPTION
**Pull request by contingency40**
- Pink Bed intro dialogue
- All entrust dialogues in the first phase for all four forms
- All five first phase loss scenes
- First phase escape dialogue
- Second phase entrust/indulge dialogue
- Second phase loss scene
- Second phase escape dialogue
- Some typo fixes found by Strate
- Other various typo fixes
- Sage's Wand weapon description

### CODE REVIEW

File: Gaijinizer patch/Troops.txt [hunk 1]
```diff
@@ -12017,10 +12017,10 @@ Do you see this red thread, \\n[1]-kun?#DL
...
 \\C[0]そういう『理由付け』で、この空間は成り立ってるの。
 > CONTEXT: Troops/298/pages/1/130/Dialogue/enemy28
-This are my \\C[2]"strings"...
-I'm using this thread to control the other Daji.....
+These are my \\C[2]"puppeteer strings"...
+I use them to control the other Daji...
 
-\\C[0]That's the reasoning behind this space.#DL
+\\C[0]That's the purpose of this space.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:12024]: Note

File: Gaijinizer patch/CommonEvents_131.txt [hunk 2]
```diff
@@ -17,18 +17,18 @@ you suck via other Daji's! ♡#DL
...
 もっと快楽に歪んだ顔を見せて…♡
 > CONTEXT: CommonEvents_131/91/Dialogue/enemy18
-It's so cute that you're struggling
-uselessly, too♡
+It's so cute that you're pointlessly
+struggling...♡
 
-Show me a more pleasured and contorted face...♡#DL
+Show more of your face perverted in pleasure...♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_131.txt:32]: Note

File: Gaijinizer patch/CommonEvents_132.txt [hunk 3]
```diff
@@ -25,10 +25,10 @@ It would seem he can't wake up anymore...#Ron
...
 心の中ではこういう結末を望んでいたんじゃないの？
 > CONTEXT: CommonEvents_132/21/Dialogue/enemy28
-...Hey, \\N[1]-kun?
+...Hey, \\N[1]-kun.
 
-I think you were really hoping in your heart
-for this to end this way, weren't you?#DL
+Was this how you were actually hoping it would
+end for you?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_132.txt:32]: Note

File: Gaijinizer patch/CommonEvents_657.txt [hunk 4]
```diff
@@ -32,10 +32,10 @@ the sound of it!#DL
...
 > CONTEXT: CommonEvents_657/133/Dialogue/enemy28
-\\n[1]-kun's energy is pouring in more
-and more... There is no dignity in any living
-thing, let alone a human being.
-I guess you wanted an ending like this...♡#DL
+More and more of your energy's pouring into her.
+Forget human dignity, there're things living
+under damp logs with more dignity than you.
+But this is the ending you I wanted I guess... ♡#RonLocalization
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_657.txt:39]: "you I wanted I guess" -> "you wanted, I guess"

File: Gaijinizer patch/CommonEvents_657.txt [hunk 5]
```diff
@@ -174,19 +176,18 @@ the scary monster ♡#DL
...
 > CONTEXT: CommonEvents_657/265/Dialogue/enemy28
-And you don't get eaten up, you get gobbled up
-and licked up forever...
-In the end, the soul is licked and melted
-like a candy ball ♡#DL
+Plus, you won't actually be completely eaten.
+You'll be sloshing around in there forever, your
+last moments will consist of your soul being
+licked and dissolved like a gobstopper. ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_657.txt:183]: Note

File: Gaijinizer patch/CommonEvents_657.txt [hunk 6]
```diff
@@ -174,19 +176,18 @@ the scary monster ♡#DL
...
 せめてエサはエサらしく、豪快に食べられちゃって♪
 > CONTEXT: CommonEvents_657/270/Dialogue/enemy28
-Now, let's not even think about wasting our
-time....
-At least the prey looks like the food,
-it's eaten with gusto♪#DL
+Now, don't bother thinking about pointlessly
+resisting, and just heroically let yourself be
+eaten up like the good little prey you are. ♪#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_657.txt:193]: Note

File: Gaijinizer patch/CommonEvents_658.txt [hunk 7]
```diff
@@ -123,10 +123,10 @@ into a position as if I was dancing.....#DL
...
 キミはもっと激しいのが好きなのかと思ってたよ
 > CONTEXT: CommonEvents_658/251/Dialogue/enemy28
-It's all right...♪ It's all right...♪
+[Waltz onomatopoeia noises]...♪ [Waltz onomatopoeia noises]...♪
 
-That's unexpected, \\n[1].
-I thought you liked it more intense.#DL
+Now this is unexpected, I thought you liked it
+a lot more rough.#CheckLater
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_658.txt:130]: Note

File: Gaijinizer patch/CommonEvents_658.txt [hunk 8]
```diff
@@ -135,10 +135,10 @@ I thought you liked it more intense.#DL
...
 > CONTEXT: CommonEvents_658/256/Dialogue/enemy28
-But even though it's a dream, when you think
-about the end of your life, it is true that one
-of the choices is to pass away peacefully and
-gently without thinking about anything.♪♡#DL
+But if you think about it in terms of this being
+your final moments, rather than this just being a
+dream, it's certainly an option to gently pass on
+not having to think about a thing. ♪ ♡#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_658.txt:142]: Note

File: Gaijinizer patch/CommonEvents_659.txt [hunk 9]
```diff
@@ -119,19 +119,17 @@ for the plants...♡#DL
 
 妾の華の、蜜壷の中で……
 > CONTEXT: CommonEvents_659/228/Dialogue/enemy18
-Good... You may indulge your mistress's body
-to the fullest... ♡
-
-In the honeypot of your mistress' flower...#DL
+That's fine... You can indulge in my body to your
+heart's content inside my flower's honeypot... ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_659.txt:124]: Line space between first and second sentences.

File: Gaijinizer patch/CommonEvents_659.txt [hunk 10]
```diff
@@ -150,12 +148,12 @@ Was the only plant type demon you met Lampas?#DL
 
 ランパスドロップが貴重なのもそのせいなんだけどね
 > CONTEXT: CommonEvents_659/257/Dialogue/enemy28
-Lampas is a 'plant' demon created by Hecate, so
-it is very different from the so-called
-'Alraune' species of the demon world.
+Lampas is a plant demon created by Hectate, so
+they're quite different from the so-called
+"Alraune" demons in the Demon Realm.
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_659.txt:153]: Hectate -> Hecate

File: Gaijinizer patch/CommonEvents_660.txt [hunk 11]
```diff
@@ -106,20 +108,20 @@ to be wrapped up like it was then...?#DL
...
 > CONTEXT: CommonEvents_660/224/Dialogue/enemy17
-I love people who are honest about their desires♡
-
-Follow your instincts, jump on them...
-I'll take it in stride♡#DL
+I love humans who are honest with their
+desires. ♡
+Follow your instincts and leap into my arms, I'll
+accept you completely. ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_660.txt:115]: Note

File: Gaijinizer patch/CommonEvents_660.txt [hunk 12]
```diff
@@ -140,10 +142,10 @@ Ours is too big for a human to see, right?#DL
...
 > CONTEXT: CommonEvents_660/254/Dialogue/enemy28
-I'm sure you've already experienced this,
-but they're so big that a big man's head
-could be buried completely...♡
-And the softness is like a marshmallow, right?#DL
+I'm sure you already know this by now, but
+they're big enough to fit a man's head
+snugly between... ♡
+And the softness is like a marshmallow, right?#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_660.txt:149]: "the softness" -> "their softness".

File: Gaijinizer patch/CommonEvents_661.txt [hunk 13]
```diff
@@ -27,10 +27,10 @@ Let's see more of that face... \\N[1]-sama♡#DL
 理性など捨て、本能のまま快楽を受け入れよ…
 決して悪いようにはせぬぞ♡♡
 > CONTEXT: CommonEvents_661/9/Dialogue/enemy18
-Your mistresses love human "lust"...♡
+The thing we love the most is human lust... ♡
 Throw away your reason, and embrace your
 instincts of pleasure...
-We won't do anything bad to you♡♡#DL
+We won't do anything bad to you... ♡♡#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_661.txt:34]: Note

File: Gaijinizer patch/CommonEvents_661.txt [hunk 14]
```diff
@@ -120,9 +120,9 @@ couldn't have been more than that♡#DL
 妾の胸に挟まれながら、無様に逝ってしまえ♡♡
 > CONTEXT: CommonEvents_661/189/Dialogue/enemy17
 Don't worry, whether you want to or not, I will
-devour all of your energy ♡
-Let him die while sanwiched between his
-mistress's breasts♡♡#DL
+devour all of your energy. ♡
+You can pathetically pass on sandwiched between
+my breasts... ♡♡#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_661.txt:126]: Note

File: Gaijinizer patch/Troops.txt [hunk 15]
```diff
@@ -12264,9 +12259,9 @@ Isn't that right, \\n[1]?#DL
 これからおヌシの身体に聞いてやる
 すでに身悶えが止まっておらぬ、この正直な身体にな……♡
 > CONTEXT: Troops/298/pages/1/358/Dialogue/enemy18
-No... you don't have to open your
-mouth to answer. I only have to ask
-your body from now on...♡#DL
+There's no need to answer. From now on your body
+will be the one doing the talking, and it hasn't
+stopped writhing in pleasure, how honest of it. ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:12265]: Note

File: Gaijinizer patch/Troops.txt [hunk 16]
```diff
@@ -12240,9 +12236,9 @@ Take a good look!#DL
 観念して、快楽に溺れてしまうがよい♡
 > CONTEXT: Troops/298/pages/1/339/Dialogue/enemy17
 Hohoho... I've caught you, \\n[1].
-He can no longer leave his mistress's body...
-It is better for him to give up and indulge
-in the pleasure ♡#DL
+You can no longer escape our body...
+You should just give in and indulge in the
+pleasure. ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:12242]: "body" to "bodies", unless they're specifically mentioning their body as one here.

File: Gaijinizer patch/CommonEvents_131.txt (hunk 2 above)
Comment (contingency40) [reply Gaijinizer patch/CommonEvents_131.txt:32]: Mind if I change "me" to "us", since that dialogue pops up when all of them are there?

Comment (Triketos) [reply Gaijinizer patch/CommonEvents_131.txt]: Definitely, sounds great 👍

**Review by Triketos — APPROVED** [review]
After the last few comments are done, we're good to go!

**Pull request merged**